### PR TITLE
feat(spec-v3): Support for OpenApi specification v3

### LINKED ISSUE #21: Support for version 3
Thoughts on providing support for version 3 of the OpenAPI specification?
https://blog.readme.io/an-example-filled-guide-to-swagger-3-2/

### PR DESCRIPTION
**Pull request by GMishx**
Check the specification version based on presense of swagger field (v2) and openapi field (v3) and parse the specification accordingly.

Closes #21

### CODE REVIEW

**Comment by GMishx** [comment]
The PR is in WIP state but can generate the HTML.

**Comment by twskj** [comment]
looks good so far thanks 👍

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
   "homepage": "https://github.com/twskj/pretty-swag#readme",
   "dependencies": {
     "esprima": "^4.0.0",
+    "expand-template": "^2.0.3",
```
Comment (twskj) [inline package.json:33]: You might not need it. Modern js can do template out of the box by using backtick
```
var name = "world";
console.log(`hello ${name}`);
```

Comment (GMishx) [reply package.json:33]: I am using it for the method you have posted here. If you see the [OpenAPI specification for server object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#serverObject), you can see that user can add place holder to the URL and provide possible values for that.

Comment (GMishx) [reply package.json:33]: I am using it here: https://github.com/twskj/pretty-swag/pull/62/files#diff-44371df60e97437c264594912e8623a7R868

**Comment by GMishx** [comment]
I have done everything I could in `pretty-swag`. There are some changes required in `livedocs` to better reflect the OpenAPI specification 3.

Please review the changes.

**Comment by twskj** [comment]
I'll make time to look at it this weekend.
Thanks for the PR

**Comment by twskj** [comment]
Can you quickly switch base branch to `development`?
Thanks!

**Comment by GMishx** [comment]
Done.

**Comment by twskj** [comment]
`livedoc` need to be updated to accommodate new things in V3. I am messing around with difference idea. I should have something out soon.

**Comment by GMishx** [comment]
Do let me know if I could help there.

**Pull request merged**